src/app/modules/group/components/group-log-view/group-log-view.component.html

```diff
@@ -35,15 +35,22 @@
           pTemplate="body"
```

Comment:
UI issue reported by @smadbe 
The text should not be larger here
![screenshot](https://user-images.githubusercontent.com/1053150/167797257-df4a6530-0518-4bbf-b98f-81de5cd37315.png)

Reply:
Fixed